src/common/wizard/Wizard.tsx

```diff
@@ -105,14 +108,35 @@ const Wizard = ({
 
   const formContentComponent = React.Children.toArray(children)[0];
 
+  const confirmGdprPrivacyPolicies = isLastFormStep(currentStep);
+  const gdprColProps = { lg: { size: 10, offset: 1 }, xl: { size: 8, offset: 2 } };
   // FIXME
   /* eslint-disable react/no-unused-prop-types */
   return (
     <Form initialValues={initialValues} onSubmit={handleSubmit}>
       {({ invalid, values }: { invalid: boolean; values: {} }) => (
-        <>
+        <div className={`${!!gdprNotes ? 'vene-form-container__without-bottom-margin' : 'vene-form-container'}`}>
```

Comment:
could be something like {`${`vene-form-container${gdprNotes && '__without-bottom-margin'}`}`}

Reply:
changed